src/components/AddLayerModal/index.tsx

```diff
@@ -152,6 +152,7 @@ export const AddLayerModal: React.FC<AddLayerModalProps> = ({
         };
         layerToAdd.set('layerConfig', layerConfig);
         targetGroup.getLayers().push(layerToAdd);
+        map.getLayers().dispatchEvent('change');
```

Comment:
Can't we reuse the existing `change:layers` event of the [root layer group](https://openlayers.org/en/latest/apidoc/module-ol_layer_Group-LayerGroup.html)?

Reply:
Yes that is correct! It will be adjusted